controller/handle-result-deletion_fuzz_test.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2019 LitmusChaos Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"testing"
+
+	fuzz "github.com/AdaLogics/go-fuzz-headers"
+)
+
+func FuzzSetResultData(f *testing.F) {
+	f.Fuzz(func(t *testing.T, data []byte) {
+		fuzzConsumer := fuzz.NewConsumer(data)
+		targetStruct := &struct {
+			resultDetails ChaosResultDetails
+		}{}
+		err := fuzzConsumer.GenerateStruct(targetStruct)
+		if err != nil {
+			return
+		}
+		targetStruct.resultDetails.setResultData()
+		if _, exist := resultStore[string(targetStruct.resultDetails.UID)]; !exist {
```

Comment:
Can we assert this error?

require.Equal(t, true, exist)

Reply:
Isn't `!exist` already doing that since the value of `exist` can be only `true` or `false` @ispeakc0de ?

Reply:
I think t.Error is equivalent to Log followed by Fail.